lte/gateway/c/core/oai/lib/mobility_client/MobilityClientAPI.hpp

```diff
@@ -20,13 +20,14 @@
 
 #include "stdint.h"
 
+#include "lte/gateway/c/core/oai/include/spgw_state.hpp"
```

Comment:
This should be after the C headers as per the [style guide](https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes)